seqsign/templates/sequence_signature.html

```diff
@@ -158,4 +158,16 @@
         //});
 
     </script>
+
+    <!-- increase browser drawing performance for displaying (larger) alignments -->
+    <script>
+      $(document).ready(function () {
+        $('.displayTableAfterLoad').css("display", "table");
+        $('.internal-scroll-div').css('width', $('.dynamic-div').outerWidth() );
+      });
+
+      $(window).on("load", function () {
+          ApplyCutoff(40);
+      });
+    </script>
```

Comment:
So why does this need to be moved from alignment.js directly into the template?

Reply:
As discussed 😄